### PR TITLE
Feat/10 second iteration of issue templates

### PR DESCRIPTION
**Pull request by signekb**
This PR includes the following changes:
- Added 's' to `attributes` key in BUG-REPORT (issue template did not show up when creating new issue bc of this error)
- Renaming of feature request and bug report templates to use hyphens (to follow example naming in [GitHub Docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms)) 
- Other minor text edits in feature request and bug report templates

### CODE REVIEW

**Review by K-Beicher — APPROVED** [review]
Looks a lot clearer.

File: .github/ISSUE_TEMPLATE/FEATURE-REQUEST.yml [hunk 1]
```diff
@@ -6,25 +6,28 @@ body:
   - type: markdown
     attributes: 
       value: | 
-        Thanks for taking the time to fill out this feature request. 
-        Please search in existing issues first to ensure this feature has not already 
+        Thanks for taking the time to fill out this feature request.
+
+        NB: Please search in existing issues first to ensure this feature has not already
```
Comment (lwjohnst86) [inline .github/ISSUE_TEMPLATE/FEATURE-REQUEST.yml:11]:
```suggestion
        Note: Please search in existing issues first to ensure this feature has not already 
```

**Review by lwjohnst86 — APPROVED** [review]
Nice!

**Review by K-Beicher — COMMENTED** [review]
I can't really comment on the code, but the text changes looks good.

**Pull request merged**